prow/cluster/starter.yaml

```diff
@@ -94,7 +94,6 @@ spec:
         app: hook
     spec:
       terminationGracePeriodSeconds: 180
-      serviceAccountName: "hook"
```

Comment:
wait a sec, hook creates prowjobs?

Reply:
yes? trigger does

Reply:
right, so it needs RBAC? ... https://github.com/kubernetes/test-infra/issues/8744